src/components/Inbox/Inbox.jsx

```diff
@@ -10,16 +13,39 @@ import AppHeader from '../AppHeader';
  */
 
 const Inbox = () => {
+  const location = useLocation();
+
+  const [showForm, setShowForm] = useState(false);
+
+  localStorage.setItem('restorePath', location.pathname);
+
   return (
     <>
-      <AppHeader />
+      <Logout />
       <section id="inbox" className="panel">
-        <div>
-          Placeholder; inbox contents will go here.
-        </div>
+        <StyledButton onClick={() => (showForm ? setShowForm(false) : setShowForm(true))}>
```

Comment:
I'm not in love with this functionality, but think it's fine for first iteration of this.

I wouldn't want the button to close the `<New Message />` but maybe have a `Cancel` button along with the `Submit` button that closes it without saving user input data or submitting.

BUT I also envision `<New Message />` being more of a modal (at least on desktop) like: 
<img width="1440" alt="Screenshot 2023-05-13 at 7 56 30 AM" src="https://github.com/codeforpdx/PASS/assets/71395931/2bc49772-48f6-493c-b6c7-cff3adf39b12">

This way the user can't focus on or do anything else besides the new message. Clicking on the greyed backdrop will cancel/close, but also both Cancel and Submit buttons.